### PR TITLE
add Big heading landing in home page

### PR DESCRIPTION
**Pull request by menos09velasco**
Checklist:

- [x] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [ ] My pull request targets the `main` branch of freeCodeCamp.
- [ ] All the files I changed are in the same world language, for example: only English changes, or only Chinese changes, etc.

Closes #XXXXX

### CODE REVIEW

**Review by naomi-lgbt — CHANGES_REQUESTED** [review]
You appear to have used version ^7.0.0 of `npm`, which includes breaking changes in the way `package-lock.json` files are handled. We are currently supporting version ^6.0.0, and the changes to the `package-lock.json` will need to be reverted.

Additionally, are the changes to the client related to an open issue or discussion? If not, please provide the context for these changes. Why are they needed?

**Comment by menosvelasco** [comment]
Hi, this is my first time contributing to open source if I have missed a step or I'm doing something wrong. please let me know, I want to learn.

**Comment by nhcarrigan** [comment]
I'll leave some more specific comments directly on the files to help clarify my concerns. 🙂

File: client/i18n/locales/english/translations.json [hunk 1]
```diff
@@ -66,8 +66,9 @@
   },
   "landing": {
     "big-heading-1": "Learn to code — for free.",
-    "big-heading-2": "Build projects.",
-    "big-heading-3": "Earn certifications.",
+    "big-heading-2": "Learn to solve problems.",
+    "big-heading-3": "Build projects.",
+    "big-heading-4": "Earn certifications.",
```
Comment (nhcarrigan) [inline client/i18n/locales/english/translations.json:71]: This change here is adding a new key to be translated through the `reacti18next` functions...

**Review by menos09velasco — COMMENTED** [review]
I added this change just because this is my first time contributing to open source

File: client/src/components/landing/components/LandingTop.js [hunk 2]
```diff
@@ -30,6 +30,7 @@ function LandingTop({ page }) {
           </h1>
           <p className='big-heading'>{t('landing.big-heading-2')}</p>
           <p className='big-heading'>{t('landing.big-heading-3')}</p>
+          <p className='big-heading'>{t('landing.big-heading-4')}</p>
```
Comment (nhcarrigan) [inline client/src/components/landing/components/LandingTop.js:33]: And the key is used here, but is not replacing any existing un-translated text. So I am confused about the addition of text to the client. Why is this being added?

File: package-lock.json [hunk 3]
```diff
@@ -1,8 +1,19712 @@
 {
   "name": "@freecodecamp/freecodecamp",
   "version": "0.0.1",
-  "lockfileVersion": 1,
+  "lockfileVersion": 2,
```
Comment (nhcarrigan) [inline package-lock.json]: lockfile Version 2 is used in `npm` versions 7 and higher - our codebase officially supports `npm` major version 6. The changes to this file should be removed from your commit.

**Review by naomi-lgbt — CHANGES_REQUESTED** [review]

**Comment by menosvelasco** [comment]
so I should remove the package-lock.json?

**Comment by nhcarrigan** [comment]
You'll want to revert the changes made to the `package-lock.json`, yes. Note that you don't want to delete the file entirely. `git` offers a few ways to do this - personally, I'd `reset` against our `main` branch and make the other two changes again. 

If needed, I can revert it for you.

**Comment by menosvelasco** [comment]
yes. go ahead and do it for me.
I'm sorry

**Comment by menosvelasco** [comment]
it would be appreciated if you could send me some resources contributing to
open source.

**Comment by nhcarrigan** [comment]
A great place to start is our [contributing documentation](https://contribute.freecodecamp.org).

**Comment by nhcarrigan** [comment]
Thank you for opening this pull request.

This is a standard message notifying you that we've reviewed your pull request and have decided not to merge it. We would welcome future pull requests from you.

Reason: The content of our landing page directly impacts our SEO - as such, we generally do not make changes without input from Quincy.

Thank you and happy coding.

**Comment by menosvelasco** [comment]
you want me to close it?
delete the branch?

**Pull request closed without merging**